Source/SteveSloth/Private/MyEnemyAIIdleState.cpp

```diff
@@ -0,0 +1,26 @@
+/****************************************************************************************
+* Copyright: SteveSloth
+* Name: Aidan MacLeod
+* Script: MyEnemyAIIdleState.cpp
+* Date: May 6. 2024
+* Description: Idle State Class for all enemies to inherit from
+* TODO:
+* Known Bugs:
+****************************************************************************************/
+
+#include "MyEnemyAIIdleState.h"
+
+void UMyEnemyAIIdleState::EnterState()
+{
+	Enemy->PlayAnimMontage(IdleAnim, 1.0f);
```

Comment:
do not include the enemy base class, it will cause circular dependencies. put all of the idle stuff within this script